setup.py

```diff
@@ -31,8 +31,11 @@
     packages=['pyoozie'],
     install_requires=[
         'enum34>=0.9.23',
-        'yattag>=1.7.2',
+        'requests>=2.12.3',
         'setuptools>=0.9',
+        'six',
+        'untangle',
```

Comment:
That's a fair question. What do we want our strategy to be? Pinning to the newest version at the time of writing is one answer, but it's pretty weak.